x/programs/rust/examples/README.md

```diff
@@ -0,0 +1,17 @@
+# Examples
+
+### amm
+
+- TODO
```

Comment:
will be updated once https://github.com/ava-labs/hypersdk/pull/942/files#diff-6052e4e22edeab88464f46b3af8d314e762449e58eda78072f1eab86b89bc35f is merged